.ci/.e2e-platforms.yaml

```
@@ -45,6 +45,11 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
     instance_type: "t3.xlarge"
     username: admin
+  macos12:
+    description: "Mac OS 12 platform"
+    image: "ami-094dfaad20f351cb7"
+    instance_type: "c6g.4xlarge"
```

MacOS type -> https://aws.amazon.com/ec2/instance-types/

```suggestion
    instance_type: "mac1.metal"
```